### PR TITLE
(873) Confirm if theres enough information in assessment

### PR DESCRIPTION
**Pull request by pezholio**
This adds a new step to confirm if there is enough information in the assessment, as well as displaying the applicant's information, so the reviewer knows who to contact if they need more information. At the moment, this only tests the happy path. The next PR will implement what happens if the user selects "No"

## Screenshots

image

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Confirm if there's enough information in assessment](https://trello.com/c/Jd5xAwWR/873-confirm-if-theres-enough-information-in-assessment)

File: server/services/userService.ts [hunk 1]
```diff
@@ -7,10 +9,19 @@ interface UserDetails {
 }
 
 export default class UserService {
-  constructor(private readonly hmppsAuthClient: HmppsAuthClient) {}
+  constructor(
+    private readonly hmppsAuthClient: HmppsAuthClient,
+    private readonly userClientFactory: RestClientBuilder<UserClient>,
+  ) {}
 
   async getUser(token: string): Promise<UserDetails> {
```
Comment (richpjames) [inline server/services/userService.ts:17]: I'm a bit confused as to which scenarios to call which method in. Is it worth renaming `getUser` to be a bit more descriptive?

File: server/views/assessments/pages/suitability-assessment/sufficientInformation.njk [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+                View guidance notes
+              </span>
+    </summary>
+    <div class="govuk-details__text">
+      <p>
+        <b>Missing information that you need to make a decision</b>
+      </p>
+      <p> If you are missing information that is essential to making a decision, request further information from the probation practitioner by clicking 'No, I need more information'.</p>
```
Comment (richpjames) [inline server/views/assessments/pages/suitability-assessment/sufficientInformation.njk:23]: Not sure if makes a difference but there's an empty space after the `<p>` here

File: server/views/assessments/pages/suitability-assessment/sufficientInformation.njk [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+  <details class="govuk-details" data-module="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+                View guidance notes
+              </span>
+    </summary>
+    <div class="govuk-details__text">
+      <p>
+        <b>Missing information that you need to make a decision</b>
```
Comment (richpjames) [inline server/views/assessments/pages/suitability-assessment/sufficientInformation.njk:21]: I suggest we use `<strong>` over `<b>` as we do elsewhere

File: server/views/assessments/pages/suitability-assessment/sufficientInformation.njk [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      </p>
+      <p> If you are missing information that is essential to making a decision, request further information from the probation practitioner by clicking 'No, I need more information'.</p>
+      <p> This information must be provided to you within 5 working days. </p>
+
+      <p>If the information is not received within 5 working days, mark the decision as 'Requested information not provided by probation practitioner' when submitting your assessment. </p>
+      <p>
+        <b>Missing information that is not required to make a decision</b>
```
Comment (richpjames) [inline server/views/assessments/pages/suitability-assessment/sufficientInformation.njk:28]: Same here with the `<b>`

**Review by richpjames — APPROVED** [review]
Looks good on the whole. I've raised a couple of queries

File: server/services/userService.ts (hunk 1 above)
Comment (pezholio) [reply server/services/userService.ts]: I've renamed it to `getActingUser`, which I think makes more sense

Comment (richpjames) [reply server/services/userService.ts]: Thanks Stu

**Pull request merged**